pytorch_lightning/plugins/training_type/single_tpu.py

```diff
@@ -63,3 +59,9 @@ def on_save(self, checkpoint: dict) -> dict:
         https://github.com/pytorch/xla/blob/master/API_GUIDE.md#saving-and-loading-xla-tensors
         """
         return move_data_to_device(checkpoint, torch.device("cpu"))
+
+    def teardown(self) -> None:
+        # TPU teardown
+        if "PT_XLA_DEBUG" in os.environ:
+            del os.environ["PT_XLA_DEBUG"]
+        return
```

Comment:
this is duplicated across single_tpu and single_device? should it only be in one spot?

Reply:
my bad, I was intended to delete this from single_device.